canonical-websites/vanillaframework.io
# Update client logos to new alignment template

## Done

- Updated client logos to use our new alignment template, so they sit on an imaginary baseline which will give logos:

  - A comparable visual weight
  - Fit in a given height / width
  - Reduces elongated or very thin logos to appear a lot smaller

## QA
- `./run`
- Visit homepage
- See that the updated client logos have comparable visual weight

## Issue / Card
- Fixes - https://github.com/canonical-websites/vanillaframework.io/issues/170
